util/log/clog.go

```diff
@@ -0,0 +1,1024 @@
+// Go support for leveled logs, analogous to https://code.google.com/p/google-clog/
```

Comment:
mostly import from clog with the exception of `InitFlags` in this file (and a bunch of renames to unexport fields and smaller changes)